Source/Immutable/Private/ImmutableModule.cpp

```diff
@@ -22,11 +28,61 @@ void FImmutableModule::StartupModule() {
 #endif
 }
 
-void FImmutableModule::ShutdownModule() {
-  // This code will execute after your module is loaded into memory; the exact
-  // timing is specified in the .uplugin file per-module This function may be
-  // called during shutdown to clean up your module.  For modules that support
-  // dynamic reloading, we call this function before unloading the module.
```

Comment:
i find this comment is useful given i'm not familiar with the shutdownmodule. might be helpful for any other team member who needs to pick up work around this?